filter.js

```diff
@@ -38,9 +41,9 @@ function Filter(inputTree, options) {
   this._destFilePathCache = Object.create(null);
 }
 
-Filter.prototype.rebuild = function() {
+Filter.prototype.build = function build() {
   var self = this;
-  var srcDir = this.inputPath;
+  var srcDir = this.inputPaths[0];
```

Comment:
should we log/error/ignore additional inputPaths? or support multiple by default?